.github/workflows/nix.yml

```diff
@@ -0,0 +1,25 @@
+name: "Nix flakes CI"
+on:
+  pull_request:
+  push:
+    branches:
+      - master
+      - main
+jobs:
+  tests:
```

Comment:
Also, name the job `Ubuntu` as that is the machine it is running on (and building for). It is doing both a build and a check, so `tests` is a bit of a misnomer.

Reply:
Please name it just `Ubuntu`, not `Build and Check Ubuntu` to make it consistent with the other CIs. Also, I don't think GItHub even allows job names that are arbitrary strings (or, at least, that would be my best guess as to why the job doesn't register).